recipe/meta.yaml

```diff
@@ -43,7 +42,7 @@ requirements:
   build:
     - toolchain
     - cmake
-    - libsodium
+    - libsodium  # [not (win and py27)]
```

Comment:
What's the difference for win & py27 here? Does this result in zeromq without CURVE support?